bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Title.java

```diff
@@ -38,6 +38,13 @@
      */
     String PN_DESIGN_DEFAULT_TYPE = "type";
 
+    /**
+     * Name of the policy property that defines whether or not the title link is disabled.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.5.0
```

Comment:
@Shivanshmathur - isn't it 12.4.0?

Reply:
I'll fix that in the next commit. Thanks.

Reply:
Fixed.